airbyte-integrations/connectors/destination-mysql/src/main/java/io/airbyte/integrations/destination/mysql/MySQLDestination.java

```diff
@@ -112,6 +112,61 @@ public JsonNode toJdbcConfig(final JsonNode config) {
     return Jsons.jsonNode(configBuilder.build());
   }
 
+  private List<String> getAdditionalParameters(final JsonNode config) {
+    final List<String> additionalParameters = new ArrayList<>();
+    addJDBCUrlParameters(config, additionalParameters);
+    if (useSSL(config)) {
+      addSSLJDBCParameters(additionalParameters);
+    }
+    return additionalParameters;
+  }
+
+  private void addJDBCUrlParameters(final JsonNode config, final List<String> additionalParameters) {
+    if (config.has(JDBC_URL_PARAMS_KEY)) {
+      final String additionalParams = config.get(JDBC_URL_PARAMS_KEY).asText();
+      if (additionalParams.isBlank()) {
+        return;
+      }
+      if (useSSL(config)) {
+        for (final String p : getSSLParameters().keySet()) {
+          if (containsParameterKey(additionalParams, p)) {
+            throw new RuntimeException("Cannot overwrite JDBC parameter " + p);
+          }
+        }
+      }
+      additionalParameters.add(additionalParams);
+    }
+  }
+
+  private boolean containsParameterKey(final String additionalParametersString, final String key) {
+    final String s = String.format("%s=", key);
+    return additionalParametersString.contains(s);
+  }
+
+  private void addSSLJDBCParameters(final List<String> additionalParameters) {
```

Comment:
Nitpick. Same for this name.

```suggestion
  private void addSslJdbcParameters(final List<String> additionalParameters) {
```

Reply:
Same issue as `addJDBCUrlParameters`. It will be more straightforward if this method returns a list instead of modifying the input argument.

Reply:
done